source/actions3/actions/AMControlMoveAction3.cpp

```diff
@@ -44,6 +47,32 @@ AMControlMoveAction3::AMControlMoveAction3(const AMControlMoveAction3 &other)
 		control_ = AMBeamline::bl()->exposedControlByInfo(*(info->controlInfo()));
 	else
 		control_ = 0;
+
+	retries_ = 1;
```

Comment:
Do we not want to set the number of retries to the ```other.retries()```?  It is the copy constructor.